### PR TITLE
FIX full memory use check testcase

### PR DESCRIPTION
**Pull request by ksangeek**
This PR is for https://github.com/rapidsai/dask-cuda/issues/136

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

File: dask_cuda/tests/test_local_cuda_cluster.py [hunk 1]
```diff
@@ -29,8 +29,10 @@ def get_visible_devices():
                     range(utils.get_n_gpus())
                 )
 
-            # Use full memory
-            assert sum(w.memory_limit for w in cluster.workers.values()) == MEMORY_LIMIT
+            # Check workers use equal portion of full memory, ignoring FP rounding difference
+            per_worker_memory = round(MEMORY_LIMIT/utils.get_n_gpus())
+            for i in range(utils.get_n_gpus()):
+                assert cluster.workers[i].memory_limit == per_worker_memory
```
Comment (pentschev) [inline dask_cuda/tests/test_local_cuda_cluster.py]: I think this approach may still cause a similar problem in some cases. I would suggest testing for some limits instead:
```python
full_mem = sum(w.memory_limit for w in cluster.workers.values())
assert full_mem >= MEMORY_LIMIT-1024 and full_mem < MEMORY_LIMIT-1024
```

Comment (mrocklin) [reply dask_cuda/tests/test_local_cuda_cluster.py]: (I think that @pentschev meant a `+` rather than a `-` in that last case.)

Comment (pentschev) [reply dask_cuda/tests/test_local_cuda_cluster.py]: Indeed, the upper limit should have a `+`, I fixed that now.

Comment (ksangeek) [reply dask_cuda/tests/test_local_cuda_cluster.py]: Thank you @pentschev for suggesting an alternative approach.
The earlier approach had worked fine, and the new one also works. I updated the PR as per your suggestion. Thanks!

Comment (pentschev) [reply dask_cuda/tests/test_local_cuda_cluster.py]: Yes, the previous approach was probably good for most cases too, but having some more relaxed condition is usually more robust.

**Comment by pentschev** [comment]
ok to test

**Comment by pentschev** [comment]
Thanks @ksangeek for your contribution! Merging.

**Pull request merged**